README.md

```diff
@@ -184,21 +184,21 @@ ssh -i emma.key root@emma2.$EMMA_DOMAIN uptime
 Create the `hosts` file see `hosts.template` for template.
 Now use ansible to verify login.
 ```
-ansible all --private-key=emma.key -u root -i hosts -m ping
+ansible all --private-key=emma.key -u root --ssh-extra-args="-o StrictHostKeyChecking=no" -i hosts -m ping
```

Comment:
The ansible command is becoming a bit large.
Maybe we start using a [./ansible.cfg](http://docs.ansible.com/ansible/intro_configuration.html) file to put the private-key and ssh-extra-arg.

What do you think?